lldb/source/Target/StackFrame.cpp

```diff
@@ -1779,18 +1779,36 @@ void StackFrame::CalculateExecutionContext(ExecutionContext &exe_ctx) {
   exe_ctx.SetContext(shared_from_this());
 }
 
+bool StackFrame::DumpUsingFormat(Stream &strm,
+                                 const FormatEntity::Entry *format,
+                                 llvm::StringRef frame_marker,
+                                 bool use_fallback_format_on_error) {
```

Comment:
I am also leaning on having the `use_fallback_format_on_error` parameter removed and letting the user decide what to do if things fail. I am open to differing opinions here if others feel differently

Reply:
fwiw I agree with Greg here.